public/radar.js

```diff
@@ -120,6 +120,7 @@ function init(h, w, data) {
 
                 radar.add(pv.Dot)
                     .event("click", function() {
+                        $('#tab-'+this.status()).click();
```

Comment:
Fixed, thx. I also changed the background colour of the highlighted technology, so it doesn't blend with the version mark.